root/static/scripts/edit/URLCleanup.js

```diff
@@ -748,6 +752,40 @@ const CLEANUPS = {
       return true;
     },
   },
+  instagram: {
+    match: [new RegExp("^(https?://)?([^/]+\\.)?instagram\\.com/", "i")],
+    type: LINK_TYPES.socialnetwork,
+    clean: function (url) {
+      var m = url.match(/^(https?:\/\/)?((?:[^\/]+\.)?instagram\.com)\/([^?#]+)(?:[?#].*)?$/);
+      if (m) {
+        var scheme = m[1] || 'https://';
```

Comment:
Yes, will do, thanks. Follow-up in a [comment to MBS-9159](https://tickets.metabrainz.org/browse/MBS-9159?focusedCommentId=41804#comment-41804).